packages/forma-36-react-components/src/components/Asset/Asset.tsx

```diff
@@ -34,75 +65,33 @@ export interface AssetProps {
   testId?: string;
 }
 
-const defaultProps: Partial<AssetProps> = {
-  type: 'image',
-  testId: 'cf-ui-asset',
-};
+export function Asset({
+  className,
+  src,
+  status,
+  testId,
+  title,
+  type,
+  ...otherProps
+}: AssetProps): React.ReactElement {
+  const classNames = cn(styles.Asset, className);
 
-export class Asset extends Component<AssetProps> {
-  static defaultProps = defaultProps;
+  // Archived images will not have a preview available
+  const asImage =
+    type && type === 'image' && (!status || status !== 'archived') && src;
```

Comment:
This condition is a bit hard to read in my opinion. Could we make it simpler?

If I understood correctly, we show a preview for when the Asset is an image. The exception is if `status === 'archived'`, right?

```jsx
// check if asset is of type image and has a src
const isImage = type === 'image' && !!src

// if status is archived, do not show preview
const showPreview = isImage && status !== 'archived'
```